app/src/main/java/org/mate/ui/EnvironmentManager.java

```diff
@@ -405,6 +406,14 @@ public void clearAppData() {
         tunnelLegacyCmd(cmd);
     }
 
+    public double matchesSurroundingColor(String packageName, String stateId, Widget widget){
+        String cmd = "surroundingColor:";
+        cmd += emulator + "_" + packageName + ":";
+        cmd += stateId + ":";
+        cmd += widget.getX1() + "," + widget.getY1() + "," + widget.getX2() + "," + widget.getY2();
+        return Double.valueOf(tunnelLegacyCmd(cmd));
```

Comment:
For new commands it would be preferred to register a new EndPoint in the MATE-server (e.g. one for everything related to Accessibility) and use that instead of adding new commands to the legacy EndPoint.